### ISSUE TITLE
Center should be updated once the map is dragged?

### ISSUE THREAD

**Issue by indianazhao** [angular-ui/angular-google-maps#1865]
## Version
2.3.3

## Test Case
http://plnkr.co/edit/tsVBgAaqLw365Vdik4au?p=preview

## Steps to reproduce
Please try to drag the map.

## Expected Behavior
The center position should be updated.

## Actual Behavior
The center seems don't change.

**Comment by davidpelayo** [comment]
Correct. [Here a forked plnkr with the version 2.3.2](http://plnkr.co/edit/xY0HyePJ1mv1Ho8BoXL3?p=preview) demonstrate how it works correctly.

Thanks for reporting.

**Comment by friedolinfoerder** [comment]
+1
